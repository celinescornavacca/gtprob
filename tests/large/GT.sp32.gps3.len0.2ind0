((((((JA0,(JA1,JA2)),((KA0,(KA1,KA2)),((LA1,LA2),LA0))),(((((TA1,TA2),TA0),((WA0,(WA1,WA2)),(((ZA1,ZA2),ZA0),(AB0,(AB1,AB2))))),(SA0,(SA1,SA2))),((IA1,IA2),IA0))),(((((CB1,CB2),CB0),((DB1,DB2),DB0)),(((MA1,MA2),MA0),((NA1,NA2),NA0))),(((QA1,QA2),QA0),((RA1,RA2),RA0)))),(((AA1,AA2),AA0),((((OA0,(OA1,OA2)),((PA1,PA2),PA0)),(DA0,(DA1,DA2))),((CA1,CA2),CA0)))),((((EA0,(EA1,EA2)),(((GA1,GA2),GA0),((HA1,HA2),HA0))),(((((UA0,(UA1,UA2)),(VA0,(VA1,VA2))),(FA0,(FA1,FA2))),(((XA1,XA2),XA0),(YA0,(YA1,YA2)))),(BA0,(BA1,BA2)))),((((EB1,EB2),EB0),(FB0,(FB1,FB2))),(BB0,(BB1,BB2)))));
