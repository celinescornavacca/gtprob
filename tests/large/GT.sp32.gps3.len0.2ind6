(((((AB1,AB2),AB0),(BB0,(BB1,BB2))),(((VA1,VA2),VA0),(((CB1,CB2),CB0),((DB1,DB2),DB0)))),(((HA0,(HA1,HA2)),(IA0,(IA1,IA2))),(((((FA1,FA2),FA0),(((YA0,(YA1,YA2)),(ZA0,(ZA1,ZA2))),((RA0,(RA1,RA2)),((SA1,SA2),SA0)))),((((KA1,KA2),KA0),((LA1,LA2),LA0)),(((NA1,NA2),NA0),(OA0,(OA1,OA2))))),((AA0,(AA1,AA2)),((((((MA1,MA2),MA0),(((EB0,(EB1,EB2)),(FB0,(FB1,FB2))),((PA1,PA2),PA0))),(((DA1,DA2),DA0),(EA0,(EA1,EA2)))),(CA0,(CA1,CA2))),(((((TA0,(TA1,TA2)),((UA1,UA2),UA0)),(GA0,(GA1,GA2))),(((JA1,JA2),JA0),(((QA1,QA2),QA0),(((WA1,WA2),WA0),(XA0,(XA1,XA2)))))),(BA0,(BA1,BA2))))))));
