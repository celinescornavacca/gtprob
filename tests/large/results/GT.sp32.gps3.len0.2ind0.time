-145.193445277
)),((KA,(KA,KA)),((LA,LA),LA))),(((((TA,TA),TA),((WA,(WA,WA)),(((ZA,ZA),ZA),(AB,(AB,AB))))),(SA,(SA,SA))),((IA,IA),IA))),