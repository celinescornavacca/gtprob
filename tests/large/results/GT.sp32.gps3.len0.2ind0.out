-145.193445277
